### PR TITLE
chore: updated city-timezones to support kyiv

### LINKED ISSUE #12224: [CAL-4073] Kyiv timezone missing
When I try to set my timezone to Europe/Kyiv I get “no options” error

IMG_2471

<sub>[CAL-4073](https://linear.app/calcom/issue/CAL-4073/kyiv-timezone-missing)</sub>

### PR DESCRIPTION
**Pull request by PeerRich**
fixes #12224

see: https://github.com/kevinroberts/city-timezones/pull/26

### CODE REVIEW

**Comment by PeerRich** [comment]
currently getting an error 

CleanShot 2024-07-26 at 14 23 52@2x

**Review by emrysal — DISMISSED** [review]

**Comment by PeerRich** [comment]
works now

File: apps/web/package.json [hunk 1]
```diff
@@ -123,7 +123,7 @@
     "react-query": "^3.39.3",
     "react-schemaorg": "^2.0.0",
     "react-select": "^5.7.0",
-    "react-timezone-select": "^1.4.0",
+    "react-timezone-select": "^3.2.7",
```
Comment (PeerRich) [inline apps/web/package.json:126]: react-timezone-select needs update since it's still expecting Kiev not Kyiv

**Review by zomars — CHANGES_REQUESTED** [review]
We have some type errors with the new version:

**Comment by PeerRich** [comment]
not sure why

**Comment by AlexHladin** [comment]
> We have some type errors with the new version:
> 
> 

@PeerRich @zomars the issue with `isMulti` is fixed. Please update `react-timezone-select` to 3.2.8.
PR where issue was resolved https://github.com/ndom91/react-timezone-select/pull/129

**Comment by AlexHladin** [comment]
Hey @PeerRich, @zomars, @anikdhabal, @Amit91848 
Would you be able to merge this PR? Once you update `react-timezone-select` to 3.2.8 the issue will be solved (https://github.com/calcom/cal.com/pull/15927#issuecomment-2326727608).

**Comment by keithwillcode** [comment]
This is also implemented in #17159

**Pull request closed without merging**